tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md

```diff
@@ -0,0 +1,265 @@
+
```

Comment:
```suggestion
```

Reply:
This blank line at the top is keeping the description from displaying on the cards on the index page